### ISSUE TITLE
Add code to allow for test FxA accounts to be removed

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/fxapom#4]
This can (apparently) be accomplished [1]  by calling:

```python
acct.clear()
client.destroy_account(acct.email, "password")
```

I suggest we add a `delete_account` method to the `FxATestAccount` class to that fixtures in `conftest.py` can be set up to delete a created account via a finalizer.

[1] https://github.com/mozilla/PyFxA/blob/master/README.rst

**Comment by davehunt** [comment]
Ooh nice! Good investigation @bobsilverberg

**Comment by bobsilverberg** [comment]
This was addressed by PR #6 and is now being superseded by issue #8 as a future enhancement.
